src/Containers.jsx

```diff
@@ -640,6 +672,21 @@ class Containers extends React.Component {
             // If there are pods to show and the generic container list is empty don't show  it at all
             if (Object.keys(partitionedContainers).length > 1 && !partitionedContainers["no-pod"].length)
                 delete partitionedContainers["no-pod"];
+
+            const prune_states = ["created", "configured", "stopped", "exited"];
+            for (const containerid of Object.keys(this.props.containers)) {
+                const container = this.props.containers[containerid];
+                // Ignore pods and running containers
+                if (!prune_states.includes(container.State) || container.Pod)
+                    continue;
+
+                unusedContainers.push({
```

Comment:
I think it's a bit weird that we calculate unusedContainers on this parent react class. Let's calculate where it's used and pass the containers property all down there. There is not much performance difference